pkg/ivd/ivd_protected_entity.go

```diff
@@ -56,6 +56,14 @@ type metadata struct {
 }
 
 func (this IVDProtectedEntity) GetDataReader(ctx context.Context) (io.ReadCloser, error) {
+	this.logger.Infof("GetDataReader called on IVD Protected Entity, %v", this.id.String())
+	this.logger.Infof("Checking vCenter connections...")
+	vcConnectionErr := this.ipetm.ReloadConfig(ctx, make(map[string]interface{}))
```

Comment:
Please make this a function instead of a bunch of copy-pasta

Reply:
Done